tripal_analysis_expression/includes/feature_heatmap_form.inc

```diff
@@ -307,7 +307,8 @@ function tripal_analysis_expression_search_features($field, $value, $support_org
   $organism = isset($_GET['organism']) ? $_GET['organism'] : NULL;
 
   $query = db_select('chado.expression_feature', 'EF');
-  $query->addField('EF', 'feature_uniquename', 'uniquename');
+  $query->join('chado.feature', 'F', 'EF.feature_id = F.feature_id');
+  $query->addField('F', 'name', 'name');
```

Comment:
This change makes sense but if we remove the `feature_uniquename` this may break someone else's site who depends on it.   I think perhaps leave that in for backwards compatibility?